webview-ui/src/utils/context-mentions.ts

```diff
@@ -1,7 +1,10 @@
 import { mentionRegex } from "../../../src/shared/context-mentions"
 import { Fzf } from "fzf"
 import { ModeConfig } from "../../../src/shared/modes"
-import * as path from "path"
+
+function getBasename(filepath: string): string {
+	return filepath.split("/").pop() || filepath
```

Comment:
Does this work for both windows and mac/linux?

Reply:
@mrubens i find a way to get os info in webview without pass it from provider